src/components/MediaMaker/Settings/VideoSettings.tsx

```diff
@@ -89,27 +70,6 @@ const VideoSettings: React.FC<Props> = ({
     removeQueryParam(Object.values(QueryParam));
   }, [dispatch, removeQueryParam, seekToBeginning]);
 
-  const onSettingsUpdate = useCallback(
```

Comment:
Is there a reason why this was moved to `src/pages/quran-media-maker/index.tsx` as that file has so many lines of code already.

Reply:
I need to invoke `setIsUpdating` from inside `onSettingsUpdate,` so I thought moving it here would be better.